models/bitcrusher/sm_hdlworkflow.m

```diff
@@ -0,0 +1,93 @@
+%--------------------------------------------------------------------------
+% HDL Workflow Script
+% Generated with MATLAB 9.6 (R2019a) at 16:46:30 on 05/09/2019
+% This script was generated using the following parameter values:
+%     Filename  : '/mnt/data/NIH/simulink_models/models/simple_gain/hdlworkflow.m'
+%     Overwrite : true
+%     Comments  : true
+%     Headers   : true
+%     DUT       : [model '\' dataplane_name]
+% To view changes after modifying the workflow, run the following command:
+% >> hWC.export('DUT',[model '\' dataplane_name]);
+%--------------------------------------------------------------------------
+
+% TODO: remove timestamps from generated file headers; this a pain for source control...
+
+%% Load the Model
+model = mp.model_abbreviation;
+dataplane_name = [model '_dataplane'];
+load_system(model);
+
+%% Model HDL Parameters
+%% Set Model 'SG' HDL parameters
+hdlset_param(model, 'CriticalPathEstimation', 'off');
+hdlset_param(model, 'GenerateHDLTestBench', 'off');
+hdlset_param(model,'HDLCodingStandardCustomizations',hdlcodingstd.IndustryCustomizations());
+hdlset_param(model, 'HDLGenerateWebview', 'on');
+hdlset_param(model, 'HDLSubsystem', [model '/' dataplane_name]);
+hdlset_param(model, 'OptimizationReport', 'on');
+hdlset_param(model, 'ResourceReport', 'on');
+hdlset_param(model, 'SynthesisTool', 'Altera QUARTUS II');
+hdlset_param(model, 'SynthesisToolChipFamily', 'Cyclone V');
+hdlset_param(model, 'SynthesisToolDeviceName', '5CSEBA6U23I7');
+hdlset_param(model, 'SynthesisToolPackageName', '');
+hdlset_param(model, 'SynthesisToolSpeedValue', '');
+hdlset_param(model, 'UseRisingEdge', 'on');
+
+% XXX: Matlab refuses to put the code anywhere except hdlsrc, no matter
+%      what directory I put here... >:(
+if isunix
+    hdlset_param(model, 'TargetDirectory', './hdlsrc');
+elseif ispc
+    hdlset_param(model, 'TargetDirectory', [mp.model_path '\hdlsrc']);
```

Comment:
I'm going to see if using [mp.model_path filesep 'hdlsrc'] works on both Windows and Linux

Reply:
Changed. Still needs to be tested on Windows